docs/notebooks/sdba-advanced.ipynb

```diff
@@ -26,9 +26,8 @@
     "    In order to help dask, one can split the processing in parts. For splitting traning and adjustment, see [the section below](#Initializing-an-Adjustment-object-from-a-training-dataset).\n",
     "\n",
     "\n",
-    "* Another massive bottleneck of parallelization of xarray is the thread-locking behaviour of some methods. It is quite difficult to isolate and avoid those locking, so one of the best workaround is to use Dask configurations with many _processes_ and few _threads_. The former do not share memory and thus are not impacted when the other is locked. However, this adds many memory transfer operations and, by experience, reduces dask's ability to parallelize some pipelines. \n",
-    " \n",
-    "   This is done usually by creating a dask client with a large `n_workers` and a small `threads_per_worker`. The last section of this notebook goes into a bit more details by suggesting a template for large-scale bias-adjustment pipelines.\n",
+    "* Another massive bottleneck of parallelization of xarray is the thread-locking behaviour of some methods. It is quite difficult to isolate and avoid those lockings, so one of the best workaround is to use Dask configurations with many _processes_ and few _threads_. The former do not share memory and thus are not impacted when a lock is acitved from a thread in another worker. However, this adds many memory transfer operations and, by experience, reduces dask's ability to parallelize some pipelines. This is done usually by creating a dask client with a large `n_workers` and a small `threads_per_worker`.\n",
```

Comment:
```suggestion
    "* Another massive bottleneck of parallelization of xarray is the thread-locking behaviour of some methods. It is quite difficult to isolate and avoid those lockings, so one of the best workaround is to use Dask configurations with many _processes_ and few _threads_. The former do not share memory and thus are not impacted when a lock is activated from a thread in another worker. However, this adds many memory transfer operations and, by experience, reduces dask's ability to parallelize some pipelines. This is done usually by creating a dask client with a large `n_workers` and a small `threads_per_worker`.\n",
```
This is done ... what is this ?

Reply:
"This" is "Dask configurations with many _processes_ and few _threads_."
I'll add an example.